# Reference origin (in lat/long degrees) of OSM in AMK (near elevator)
# The latitude and longitude should be set the same here and for map generation
ref_latitude: 50.1363486
ref_longitude: 8.6474024
building: AMK
